frontend/app/components/redesign/components/WidgetPositionSelector.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { cx } from 'class-variance-authority'
+import { OptionSelector, type Option } from './OptionSelector'
+import { WIDGET_POSITION, type WidgetPositionKey } from '@shared/types'
+import { SVGPositionLeft, SVGPositionRight } from '@/assets'
+
+export interface WidgetPositionSelectorProps {
+  defaultValue?: WidgetPositionKey
+  onChange?: (value: WidgetPositionKey) => void
+  className?: string
+}
+
+const widgetPositionOptions: Option<WidgetPositionKey>[] = [
+  {
+    id: 'position-left',
+    label: 'Left',
+    value: WIDGET_POSITION.Left,
+    icon: <SVGPositionLeft className="w-11 h-11" />
+  },
+  {
+    id: 'position-right',
+    label: 'Right',
+    value: WIDGET_POSITION.Right,
+    icon: <SVGPositionRight className="w-11 h-11" />
+  },
+  {
+    id: 'position-empty',
+    label: '',
+    value: WIDGET_POSITION.Right,
```

Comment:
I think there should be a `Empty` position instead of subtly setting it to right?